COLLABORATOR_GUIDE.md

```diff
@@ -75,47 +77,86 @@ Collaborators or additional evidence that the issue has relevance, the
 issue may be closed. Remember that issues can always be re-opened if
 necessary.
 
+### Ready to land pull requests
+
+A pull request that is still awaiting the minimum review time is considered
+`ready` as soon as the CI got kicked off, it got at least one approval and
+it has no outstanding review comments. Please always make sure to add the
+appropriate `ready` label to the PR in that case and remove it again as soon
+as that condition is not met anymore.
+
+### Handling own pull requests
+
+If you as a collaborator open a pull request, please always start a CI right
+after (see [testing and CI](#testing-and-ci) for further information on how to
+do that) and post the link to it as well. Please also make sure that you start a
+new CI after each update (due to e.g., a change request in a review or due to
+rebasing).
+
+As soon as the PR is ready to land, please go ahead and do so on your own.
+Landing own pull requests distributes the work load for each collaborator
+equally. If it is still awaiting the
+[minimum time to land](#waiting-for-approvals), please add the `ready` label to
+it so it is obvious that the PR can land as soon as the time ends.
+
 ## Accepting Modifications
 
-All modifications to the Node.js code and documentation should be
-performed via GitHub pull requests, including modifications by
-Collaborators and TSC members. A pull request must be reviewed, and usually
-must also be tested with CI, before being landed into the codebase.
+All modifications to the Node.js code and documentation should be performed via
+GitHub pull requests, including modifications by Collaborators and TSC members.
+A pull request must be reviewed, and must also be tested with CI, before being
+landed into the codebase. There may be exception to the latter.
 
-### Code Reviews and Consensus Seeking
+### Code Reviews
 
 All pull requests must be reviewed and accepted by a Collaborator with
 sufficient expertise who is able to take full responsibility for the
 change. In the case of pull requests proposed by an existing
 Collaborator, an additional Collaborator is required for sign-off.
 
-In some cases, it may be necessary to summon a qualified Collaborator
-or a GitHub team to a pull request for review by @-mention.
-[See "Who to CC in issues"](./doc/onboarding-extras.md#who-to-cc-in-issues)
+In some cases, it may be necessary to summon a GitHub team to a pull request for
+review by @-mention.
+[See "Who to CC in issues"](./doc/onboarding-extras.md#who-to-cc-in-issues).
 
 If you are unsure about the modification and are not prepared to take
 full responsibility for the change, defer to another Collaborator.
 
+If you are the first collaborator to approve a pull request that has no CI yet,
+please start one (see [testing and CI](#testing-and-ci) for further information
+on how to do that) and post the link to the CI in the PR. Please also start a
+new CI in case a code change was done due to a addressed review comment or a
```

Comment:
This line is a bit unclear to me. If nothing else: `a addressed` -> `an addressed`